rebar.config

```diff
@@ -32,10 +34,6 @@
 	     [{relx, [{dev_mode, false},
 		      {include_erts, true},
 		      {include_src, false}
-		     ]},
-	      {deps, [
-		      {recon, "2.5.1"},
```

Comment:
`recon` and `observer_cli` is need only for production, it no make sense to move those deps into top level of all deps.

Reply:
"Add recon and observer_cli to the default deps: This allows to
have them also in test/experimental builds"

and no, they are also helpful when operating in a test/experimental build.

Reply:
I see, then https://github.com/travelping/capwap/pull/55#discussion_r854005778 :upside_down_face: